### PR TITLE
Logging clean-up [B-03288]

### PR DESCRIPTION
**Pull request by mjbrisebois**
### Acceptance Criteria
- every change of state has a log message
- log messages are clear and easy to understand
- log messages are easily readable (not too verbose)

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -431,23 +458,22 @@ class Envoy {
...
 	    
 	    try {
-		req_log			= await this.logServiceRequest( hha_hash, agent_id, payload, service_signature );
-		log.debug( req_log );
-
-		req_log_hash	    	= req_log;
-		log.debug("Request log commit hash: %s", req_log_hash );
+		log.debug("Log service request (%s) from Agent (%s)", service_signature, agent_id );
+		req_log_hash		= await this.logServiceRequest( hha_hash, agent_id, payload, service_signature );
+		log.info("Service request log hash: %s", req_log_hash );
```
Comment (zippy) [inline src/index.ts:472]: why info instead of debug?

**Review by zippy — COMMENTED** [review]
Logging cleanup look good.  One general question I have is the logic about what gets logged as info, and what as debug.  There's a couple places where things were changed from debug to info, which makes me think that logs will get to noisy.  I noted one place as an example where we log the hash of service request.  Which is kind of funny because it means we are logging the logging!  I think that makes sense for debugging, but not necessarily for info level logging?

File: src/index.ts (hunk 1 above)
Comment (mjbrisebois) [reply src/index.ts:486]: My rule of thumb when choosing is basically; if it is logging input right before a function call, then it is likely debug or silly; if it is logging the result of some function call, then normal or info.

My expectation is that with the "info" logging level, we should see "what" happened and with the higher levels we should see "why" it happened.

cc @zippy

File: rpc-websocket-wrappers/client.js [hunk 2]
```diff
@@ -12,24 +12,26 @@ const { TimeoutError }			= async_with_timeout;
...
 	    return;
+	}
 
 	return async_with_timeout(() => {
 	    return new Promise((f,r) => {
-		log.silly("Waiting for open event on %s", this.name );
-
 		const tid		= setTimeout(() => {
+		    log.silly("Checking ready state for RPC WebSocket client (%s): %s", this.name, this.socket.readyState );
 		    if ( this.socket.readyState === 1 )
```
Comment (JettTech) [inline rpc-websocket-wrappers/client.js:26]: Do we still need this check here as we are now returning checking and returning based on this same condition prior to entering async_with_timeout? (lines 17-20 above)

**Review by JettTech — APPROVED** [review]
Reviewed and tested.  All tests in debug mode pass and return as expected in the terminal.

File: rpc-websocket-wrappers/client.js (hunk 2 above)
Comment (mjbrisebois) [reply rpc-websocket-wrappers/client.js:26]: Yes, this check happens at a later time so the `readyState` could be different.  This timeout is just a final last check before raising the timeout error

**Pull request merged**